### PR TITLE
Add GlobeMap view option to campaign maps

### PR DESCRIPTION
**Pull request by willemarcel**
Experimentation around Globe Map view using Deck GL (#677)

### CODE REVIEW

**Comment by LanesGood** [comment]
@willemarcel nice work! I like the toggle button, and the deckgl implementation is very smooth.

I think we can treat this as a starting point. I think the next steps to explore would be:
- use same basemap styling for 2d and 3d maps
- add stationary platforms
- style the flight paths and stationary platforms according to the colors and icons in the 2d map
- Add in legend + toggle functionality

Should we make this a draft pr and keep this as a base to add in further features?

**Comment by willemarcel** [comment]
@LanesGood Thanks for the suggestion of using the same map style. I tried it before, but didn't look well. However, I decided to try it again and noted that in my first try it was missing to set the right tile size. Now it's looking great to me! 

image

We yet have that hole close to the poles, but maybe we can fix it by finding the correct colour for the `SimpleMeshLayer`. I think the tile layer overlays it and results in a darker colour.

I'll work on the other suggestions later today.

**Comment by willemarcel** [comment]
@LanesGood I've added the legend to the Globe view map and enabled the filtering by deployment and platforms. I think we can merge it after a final review.

**Review by LanesGood — APPROVED** [review]
@willemarcel nice work, this looks great!
image

File: src/components/timeline/map-view-control.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  bottom: 0px;
+  margin-bottom: 5px;
+  font-weight: ${props => (props.checked ? 600 : 400)};
+  font-family: "Titillium Web", sans-serif;
+`
+
+export const MapViewControl = ({ onClick, is3DViewEnabled }) => (
+  <ControlBox>
+    <Button action={onClick} mode={POSITIVE} noBorder>
+      Switch to {is3DViewEnabled ? "2D" : "3D"} map view
```
Comment (LanesGood) [inline src/components/timeline/map-view-control.js]:
```suggestion
      View in {is3DViewEnabled ? "2D" : "3D"}
```

**Comment by LanesGood** [comment]
@willemarcel the changes and updates look good! Made one small suggestion to shorten the button text.

**Comment by willemarcel** [comment]
@LanesGood Let me know if the latest changes I've pushed are good, so we can merge and deploy it.

**Comment by LanesGood** [comment]
@willemarcel yes, looks good to merge!

**Pull request merged**